js_modules/dagit/packages/core/src/app/ExecutionSessionStorage.tsx

```diff
@@ -163,7 +153,7 @@ current localStorage namespace in memory (in _data above) and React keeps a simp
 version flag it can use to trigger a re-render after changes are saved, so changing
 namespaces changes the returned data immediately.
 */
-export function useStorage(
+export function useExecutionSessionStorage(
```

Comment:
Renaming this out of the way - it's exported and has a generic sounding name which I didn't want to be confused for the new (actually more generic) hook